loadtest/control/simulcontroller/actions.go

```diff
@@ -1126,6 +1126,9 @@ func searchPosts(u user.User) control.UserActionResponse {
 			if err != nil {
 				return control.UserActionResponse{Err: control.NewUserError(err)}
 			}
+			if err := getProfileImageForUsers(u, keys(users)); err != nil {
```

Comment:
Unrelated but did we ever review the TODOs above? Noting that we are still living with some arbitrary choices which eventually will affect the output rates.

Reply:
Don't think we did. I guess the challenge here would be that we cannot directly count the number of mentions because that wouldn't be allowed. What we might do is look for indirect things like rates for `autocompleteUsersInTeam` and other similar things to give a rough idea.